ydb/core/tx/schemeshard/schemeshard__operation_create_sequence.cpp

```diff
@@ -246,6 +246,63 @@ class TPropose: public TSubOperationState {
     }
 };
 
+NKikimrSchemeOp::TSequenceDescription FillSequenceDescription(const NKikimrSchemeOp::TSequenceDescription& descr) {
+    NKikimrSchemeOp::TSequenceDescription result = descr;
+
+    i64 increment = 0;
+    if (result.HasIncrement()) {
+        increment = result.GetIncrement();
+    }
+    if (increment == 0) {
+        increment = 1;
+    }
+    result.SetIncrement(increment);
+
+    i64 minValue = 1;
+    i64 maxValue = Max<i64>();
+    if (increment < 0) {
+        maxValue = -1;
+        minValue = Min<i64>();
+    }
+
```

Comment:
Why this arithmagic is necessary in the PR about adding the alter sequence op?

And perhaps it deserves a line or two with the explanation in the code.

Reply:
Before that, the sequence scheme was not used and was not filled with default values, now it was necessary to add logic for filling the scheme with default values

comment added